plugins/package-managers/bundler/src/main/kotlin/Bundler.kt

```diff
@@ -404,7 +404,7 @@ class Bundler(
     }
 }
 
-internal data class GemSpec(
+internal data class GemInfo(
```

Comment:
I believe this commit should clarify which data model the class resembles. Do you have a link which could be added as KDoc here?

Reply:
This resembles nothing official. It's a custom ORT class for passing data around. That's also why is can contain ORT-only types like `VcsInfo` and `RemoteArtifact`.